tests/suite/test_transport_server_status.py

```diff
@@ -0,0 +1,114 @@
+import pytest
+
+from suite.resources_utils import wait_before_test
+from suite.custom_resources_utils import (
+    read_v1alpha1_crd,
+    patch_transport_server_from_yaml,
+)
+from settings import TEST_DATA
+
+
+@pytest.mark.ts
+@pytest.mark.parametrize(
+    "crd_ingress_controller, transport_server_setup",
+    [
+        (
+            {
+                "type": "complete",
+                "extra_args":
+                    [
+                        "-enable-custom-resources",
```

Comment:
because status reporting depends on the replica being the leader, and leader election can take time, I wonder if you had any flakes or problems when running the new tests? Note that in test_virtual_server_status.py,  leader election is disabled `-enable-leader-election=false`

Reply:
I've had no issues locally but that doesn't mean it isn't a problem. Do you suggest disabling leader election?

Reply:
@soneillf5 I'd suggest disabling for now, I've had issues in pipeline when enabled but not locally.

Reply:
updated to add `-enable-leader-election=false`